packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -768,12 +811,18 @@ abstract class SliderTickMarkShape {
 /// All shapes are painted to the same canvas and ordering is important.
 /// The overlay is painted first, then the value indicator, then the thumb.
 ///
+/// The thumb painting can also be skipped by specifying [noThumb].
+///
+/// The overlay painting can also be skipped by specifying [noOverlay].
```

Comment:
can also be => can be

... by specifying [noThumb] for [SliderThemeData.overlayShape].